gradle.properties

```diff
@@ -1,21 +1,5 @@
-# Project-wide Gradle settings.
-
-# IDE (e.g. Android Studio) users:
-# Settings specified in this file will override any Gradle settings
-# configured through the IDE.
-
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-# Default value: -Xmx10248m -XX:MaxPermSize=256m
 org.gradle.jvmargs=-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
-
 android.useAndroidX=true
-android.enableJetifier=false
\ No newline at end of file
+android.enableJetifier=false
```

Comment:
This is false by default, maybe we can simply remove it.

Also, have you thought about turning on the [build cache](https://docs.gradle.org/current/userguide/build_cache.html)?

Reply:
I'll consider it both. But not a concern for this PR.